deploy/kube/README.md

```diff
@@ -0,0 +1,77 @@
+# Istio Mixer K8s Configuration
+
+These configurations stand up an instance of the mixer, a Prometheus instance that scapes metrics the mixer exposes, and
+a Grafana instance to render those metrics. These are intended for a demo, not a real production setup. The easiest way
+to stand up these deployments is to run (from the Istio mixer root directory):
+
+    $ kubectl create configmap testdata-config --from-file=testdata/globalconfig.yml --from-file=testdata/serviceconfig.yml --from-file=testdata/prometheus.yaml
+    $ kubectl apply -f ./deploy/kube/
+    
+If you're using something like [minikube](https://github.com/kubernetes/minikube) to test locally, you can call the
+mixer server with our test client (`mixc`), providing some attribute values:
+
+    $ MIXS=$(minikube service mixer --url --format "{{.IP}}:{{.Port}}")
+    $ bazel run //cmd/client:mixc -- check -m $MIXS -a source.name=source,target.name=target,api.name=myapi,response.code=200,response.latency=100,client.id=$USER
```

Comment:
no bazel stuff in the demo, please. maybe you should distribute your client as a docker image?